### PR TITLE
Release 0.30.0

### PR DESCRIPTION
**Pull request by bcardiff**
There are still a couple of PRs in the pipeline but most of the changelog can be written.

Preview at https://github.com/bcardiff/crystal/blob/changelog/0.30.0/CHANGELOG.md

### CODE REVIEW

**Comment by asterite** [comment]
I don't know if the Changelog is still not up to date with the last PRs merged, but #7956 was merged and it's a big breaking change. But it's a good one 😅

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,116 @@
...
+- Add `alias`/`merge` methods to `YAML::Builder` and `YAML::Nodes::Builder`. ([#7949](https://github.com/crystal-lang/crystal/pull/7949), thanks @Blacksmoke16)
+
+### Files
+
+- Adds `File.readlink` to match `File.symlink`. ([#7858](https://github.com/crystal-lang/crystal/pull/7858), thanks @didactic-drunk)
+
+### Networking
+
+- **(breaking-change)** Improve URL encoding. `URI.escape` and `URI.unescape` are renamed to `URI.encode_www_form` and
+`URI.decode_www_form`. Add `URI.encode` and `URI.decode`. ([#7997](https://github.com/crystal-lang/crystal/pull/7997), thanks @straight-shoota)
```
Comment (Sija) [inline CHANGELOG.md]: This should be joined with previous line.

Comment (bcardiff) [reply CHANGELOG.md]: Yup. Thanks

**Comment by RX14** [comment]
(whoops wrong button)

**Pull request merged**